apps/passport-client/src/subscribe.ts

```diff
@@ -0,0 +1,53 @@
+import {
```

Comment:
this file is the internals of how components are able to subscribe to changes of particular fields on the global state object